src/ZuluSCSI_disk.h

```diff
@@ -93,6 +101,9 @@ void scsiDiskCloseSDCardImages();
 bool scsiDiskOpenHDDImage(int target_idx, const char *filename, int scsi_id, int scsi_lun, int blocksize, S2S_CFG_TYPE type = S2S_CFG_FIXED);
 void scsiDiskLoadConfig(int target_idx);
 
+// During loading, checks if a filename is appropriate for further processing as a disk image
```

Comment:
Comment could have a bit more detail: checks file extension for ignored file types. Does not check file name prefix.

Reply:
I added some additional language explaining the function. The name was given with the thought that it may be expanded in the future if there are additional things that should cause a file to be ignored, but I agree that it currently is a bit misleading.